hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/db.json

```diff
@@ -1,15 +1,15 @@
 {
   "clusterState": {
+    "pipelines": 32,
     "totalDatanodes": 24,
     "healthyDatanodes": 24,
-    "pipelines": 32,
     "storageReport": {
-      "capacity": 32985348833280,
-      "used": 15942918602752,
-      "remaining": 12094627905536
```

Comment:
Can I ask why did you change this section?

Reply:
I have copied actual end point Response in DB.json so it will be same.